web_edge_page_data.py

```diff
@@ -297,6 +297,7 @@ def _get_heatmap_data(self, index_sample_scores, expression_matrix):
                                  keep_n_samples=keep_n_samples,
                                  keep_n_genes=keep_n_genes)
 
+    print("Started PathCORE edge data insertion operations")
```

Comment:
Absolutely!! I need to implement logging across a lot of the PathCORE-related repos. It is on my to-do list. :D thanks for making this suggestion!